### PR TITLE
Support multi-channel setups

### PR DESCRIPTION
**Pull request by AurelienGasser**
Companion PR: https://github.com/SubstraFoundation/hlf-k8s/pull/75 & https://github.com/SubstraFoundation/hlf-k8s/pull/84

### CODE REVIEW

**Comment by thibaultrobert** [comment]
To test the multi-channel behavior: 
- See the [companion PR](https://github.com/SubstraFoundation/hlf-k8s/pull/75) to set up a multi-channel hlf network
- Uncomment the second channel material (or revert [this commit](https://github.com/SubstraFoundation/substra-backend/pull/284/commits/2b96db36a3ab02159c1bc4b29c0b49ef5e0dd3d7))
- Launch substra tests twice, once with each pair of users from the same channel

File: skaffold.yaml [hunk 1]
```diff
@@ -210,6 +210,9 @@ deploy:
               value: "true"
             - name: DEFAULT_THROTTLE_RATES
               value: '120'
+          channels:
```
Comment (Kelvin-M) [inline skaffold.yaml:213]: You missed org-1 :)

Comment (thibaultrobert) [reply skaffold.yaml:216]: Fixed.

**Comment by Kelvin-M** [comment]
Closed in favor of #300 as #299 has been merged :)

**Pull request closed without merging**